README.md

```diff
@@ -241,6 +242,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [LimeChat](http://limechat.net/mac/)
   - [LittleSnitch](http://www.obdev.at/products/littlesnitch/)
   - [Livestreamer](http://livestreamer.tanuki.se/)
+  - [LSColors](https://github.com/trapd00r/LS_COLORS)
```

Comment:
just remove it, and use a custom config =)
https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-or-any-file-or-directory